static/css/bundle.css

```diff
@@ -68,6 +68,10 @@ h1, h2, h3, h4, h5 {
 
 h1 {
   font-size: 280%; }
+h1 a:hover{
```

Comment:
Don't edit the compiled css file. You should edit the scss files and run gulp to generate the bundle.

Reply:
i don't know how to gulp, i do have prepros however. i keep trying to run gulp and i all it get is a Assertion error by typing 'gulp default' in the directory where the gulpfile.js is.